_guides/change-logs.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Writing Good Change Logs
+layout: default
+category: Documentation
+summary: A layout for how to write consistent and informative change logs that are meaningful to those reading them.
+date: 2020-02-25
+---
+
+{{ page.summary }}
+
+## Background
+
+Having a good changelog can be critical for those using your applications and services.
+It lets them know what has changed, and what they need to change.
+Ensuring it stays consistent and is clear helps those understand the changes better.
+Notes should be easy to read and convey a clear message to the reader.
+
+## General logs
```

Comment:
Should it be stated to *keep your releases small* so as to keep your list of categories easier to read?

Reply:
I feel like that isn't as relevant, if the change log is well detailed it shouldn't matter how _big_ it is.